.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
+npm test
```

Comment:
Don't change files that have nothing to do with this PR. If this is a necessary change, create a new PR only with this changes and we will decide if we want to merge it.